applications/machine_learning/configuration/thingy52_nrf52832/led_state_def.h

```diff
@@ -37,22 +37,22 @@ static const struct led_effect ei_data_forwarder_led_effects[] = {
 static const struct ml_result_led_effect ml_result_led_effects[] = {
 	{
 		.label = NULL,
-		.effect = LED_EFFECT_LED_BLINK(300, LED_COLOR(255, 255, 255)),
+		.effect = LED_EFFECT_LED_BLINK(200, LED_COLOR(255, 255, 255)),
+	},
+	{
+		.label = ANOMALY_LABEL,
+		.effect = LED_EFFECT_LED_ON_GO_OFF(LED_COLOR(255, 0, 0), 2500, 500),
```

Comment:
Could you change 3 gestures to RGB and leave magneta for anomaly? I understand R and could be undestrood as error by from the other hand it could be more intuitive for users to expect RGB for 3 gestures we have. I would personally prefer this.

Reply:
ok